Directory.Build.targets

```diff
@@ -0,0 +1,2 @@
+<Project>
```

Comment:
Yes, visibility. And it prevents the traversal search for D.B.targets from looking any higher than the repo root.

Plus, this will have content soon... (lineups) :)